### PR TITLE
Anti aim assist, Anti player invisible & Detecting status of resources

### PR DESCRIPTION
**Pull request by Robi321**

### CODE REVIEW

File: ac_c.lua [hunk 1]
```diff
@@ -56,7 +56,7 @@ end)
 
 Citizen.CreateThread(function()
 	while true do
-		Citizen.Wait(60000)
+		Citizen.Wait(30000)
```
Comment (Blumlaut) [inline ac_c.lua:59]: why?

File: ac_c.lua [hunk 2]
```diff
@@ -72,11 +72,18 @@ Citizen.CreateThread(function()
 		if GetUsingnightvision() then
 			TriggerServerEvent("AntiCheese:Night")
 		end
-		
-		local playerPed = PlayerPedId()
```
Comment (Blumlaut) [inline ac_c.lua:76]: why?

File: ac_c.lua [hunk 3]
```diff
@@ -72,11 +72,18 @@ Citizen.CreateThread(function()
...
+
+		if IsPedSittingInAnyVehicle(PlayerPedId()) and IsVehicleVisible(GetVehiclePedIsIn(PlayerPedId(), false)) then
 			TriggerServerEvent("AntiCheese:CarVisible")
 		end
+
+		if not IsEntityVisible(PlayerPedId()) then
+			TriggerServerEvent("AntiCheese:PlayerVisible")
+		end
+
+		if GetLocalPlayerAimState() ~= 3 and not IsPedInAnyVehicle(PlayerPedId()) then
```
Comment (Blumlaut) [inline ac_c.lua:84]: isnt this a setting in GTA? I dont think this is sane to list as a cheat, unless you cant actually change this setting in FiveM

File: ac_c.lua [hunk 4]
```diff
@@ -239,6 +246,14 @@ for i, event in pairs(negativePayEvents) do
 	AddEventHandler(event, negativePayFunc)
 end
 
+AddEventHandler("onClientResourceStart", function(resourceName)
```
Comment (Blumlaut) [inline ac_c.lua:249]: can this trigger if a player is joining? i never used onClientResourceStart, also, would this trigger on an anticheese restart?

Comment (Jaccosf) [reply ac_c.lua:249]: This triggers when someone starts a resource from a client console, if I'm not wrong.

Comment (Jaccosf) [reply ac_c.lua:249]: So yes. this would also trigger when anticheese is being restarted (from the client)

File: ac_c.lua (hunk 2 above)
Comment (Jaccosf) [reply ac_c.lua:76]: Yes, indeed, why would you unoptimise it by not localizing `playerPed`?

**Pull request closed without merging**